relayer-engine/src/listener/spyEventSource.ts

```diff
@@ -53,6 +54,11 @@ export async function createSpyEventSource(
   });
 }
 
+const spyConnectionsGauge = new Gauge({
```

Comment:
I think at some point it'll be useful to put all metrics somewhere

Reply:
we have been separating them in their own file for each section. I haven't done that for this one because it's just one and it seemed simple enough to see exactly what the metric is closed to where its used but I'm open to moving it.

Reply:
I think it's not about how simple it is, but about having a place in which you can inspect the code and see what metrics you have available and what data they contain... think of someone new to the project trying to create or update a metric

Reply:
yeah I agree with @solanoepalacio